fixed index out of bounds in recognizeImage method

whiskermrr opened the pull request:
While trying to run model with 2 classes i've been getting index out of bounds in label list. Sometimes array outputClasses may contains negative values. It happens only with custom models that have small amount of labels.

whiskermrr commented:
I signed it!

jdduke commented on lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/tflite/TFLiteObjectDetectionAPIModel.java:
```diff
@@ -207,12 +207,15 @@ public static Classifier create(
-              outputScores[0][i],
-              detection));
+      int labelIndex = (int) outputClasses[0][i] + labelOffset;
+      if(labelIndex >= 0 && labelIndex < labels.size()) {
```
This seems like a bug in the model? In any case, we should log an error in this case with some details about the labelIndex and score.

Nit: space after if ("if (labelIndex >= 0...)").

MarkDaoust commented:
@whiskermrr can you resolve the comments from the reviewer?

whiskermrr commented:
Sorry for my late response. The problem was with model. Probably because I've changed output_arrays while converting model to tflite. Newer models working great and I've never faced this problem again. I deleted tflite file that was causing this error so I'm not able to reproduce it.
